### DISCUSSION TITLE
Use RNR (React Native Reusables) components

### DISCUSSION THREAD

**Discussion by joshuadutton** [obytes/react-native-template-obytes#419] [Ideas]
We are using RNR in some of our projects, and I'm curious what you think of it as a potential addition to your starter.

**Comment by yjose** [comment]
Hey @joshuadutton, I recently tried using React Native Reusable for a project, and it seems to be a good choice. We made a small change to our project structure for people want to use it. as we [move the ui folder inside components ](https://github.com/obytes/react-native-template-obytes/commit/f113fa0174de9e9f62ae09b41b4edee3170f3101)

My concern at the moment is that RNR have some file that are uppercase and do not follow our file name convention. 
I already started a discussion a few weeks ago, but I didn't have the time to follow up with the maintainer.
Let me know, based on your experience, why we should use it? and will make sure to follow up when ever i have some time

**Reply by joshuadutton** [reply]
These are the benefits I see:
- they already have a few more components than you do
- it falls back to radix-ui on web
- it's easy to pull components in as you need them with full customization

As we use more AI generation with base components and tailwind, though, we find we need fewer components in our base library and may not be needing as many components anyway.

We are working on project right now evaluating which RNR components we will pull in. To your point, the Obytes starter project structure makes this easy.
